Lib/ntpath.py

```diff
@@ -146,29 +146,55 @@ def splitdrive(p):
             sep = b'\\'
             altsep = b'/'
             colon = b':'
+            prefix = b'\\\\?\\'
+            unc_prefix = b'UNC\\'
+            vol_prefix = b'VOLUME'
```

Comment:
```suggestion
            vol_prefix = b'VOLUME{'
```

Reply:
FYI, the C macro that checks for a volume name checks this part, and also the four hyphens, and the final "}" bracket. Unlike `PathCchSkipRoot()`, the macro does not validate that the 32 characters in the GUID are hex digits (0-9A-Fa-f). Checking for "Volume{........-....-....-....-............}" is considered good enough.